### PR TITLE
Moving files from MePo theme Titan skin with FanartHandler support

### PR DESCRIPTION
**Pull request by KJ-Cox**
Moving files from MePo theme Titan skin with FanartHandler support

### CODE REVIEW

File: mediaportal/MediaPortal.Base/skin/Titan/DialogPinCode.xml [hunk 1]
```diff
@@ -1,145 +1,134 @@
...
-        <import>common.dialog.xml</import>
-        <control>
-            <id>1</id>
-            <description>Lock</description>
-            <type>image</type>
-            <posX>850</posX>
-            <posY>266</posY>
-            <width>48</width>
-            <height>48</height>
-            <texture>lock.png</texture>
```
Comment (andrewjswan) [inline mediaportal/MediaPortal.Base/skin/Titan/DialogPinCode.xml:16]: Why did we remove this picture?

File: mediaportal/MediaPortal.Base/skin/Titan/mytvrecordedtv.xml [hunk 2]
```diff
@@ -6,7 +6,6 @@
 <define>#header.label:604</define>
 <controls>
 
-    <import>common.default.background.xml</import>
```
Comment (andrewjswan) [inline mediaportal/MediaPortal.Base/skin/Titan/mytvrecordedtv.xml:9]: Don't we need background here?

File: mediaportal/MediaPortal.Base/skin/Titan/DialogPinCode.xml (hunk 1 above)
Comment (KJ-Cox) [reply mediaportal/MediaPortal.Base/skin/Titan/DialogPinCode.xml:16]: This was copied from MovingPictures and looks better.

File: mediaportal/MediaPortal.Base/skin/Titan/mytvrecordedtv.xml (hunk 2 above)
Comment (KJ-Cox) [reply mediaportal/MediaPortal.Base/skin/Titan/mytvrecordedtv.xml:9]: Probably, I just copied this over from MePo but did not check it. I personally have not made any changes to this as I don't use the plugin.

**Pull request merged**